guides/common/modules/proc_performing-post-upgrade-tasks-on-your-project-server.adoc

```diff
@@ -0,0 +1,19 @@
+[id="performing-post-upgrade-tasks-on-your-{project-context}-server_{context}"]
```

Comment:
```suggestion
[id="performing-post-upgrade-tasks-on-your-{project-context}-server"]
```

If we only include the procedure once per guide, we could do without "{context}" in the anchor.